languages/swift/exercises/concept/tuples/.docs/hints.md

```diff
@@ -0,0 +1,17 @@
+## General
+
+- The individual components of a tuple can be accessed by appending a _.n_ to the name of the tuple, where _n_ is the index of the component, starting with 0.
+
+## 1. Convert coordinates to polar
+
+- The individual components of a labeled tuple can also be accessed by appending a _.label_ to the name of the tuple, where _label_ is the label name of the component.
+- The `atan2(_:,_:)`function requires `Foundation` to be imported.
+- Floating point types in Swift have the [squareRoot() method][squareroot-docs].
```

Comment:
Could you maybe rephrase a bit not to give away the exact solution (as in not mentioning `atan2` or `squareRoot`)?